homeassistant/components/reolink/select.py

```diff
@@ -132,6 +157,42 @@ def _get_quick_reply_id(api: Host, ch: int, mess: str) -> int:
     ),
 )
 
+CHIME_SELECT_ENTITIES = (
+    ReolinkChimeSelectEntityDescription(
+        key="play_tone",
+        translation_key="play_tone",
+        get_options=_get_chime_play_list(),
+        method=lambda chime, name: chime.play(ChimeToneEnum[name].value),
+    ),
```

Comment:
Using a select gives a much nicer user interface where a user can just select the chime to play.
(I could have used button entities, but then you get 9 button entities instead of 1 select).
The value is set to None always and when selecting a option, it will play the ringtone and also emedialtly go back to/stay on the None state (async_write_ha_state).

I have tested this and you can select the same ringtone again to play it a second time.
Also you can use the select.select_option to play the same ringtone muliple times after eachother (you need a delay of the time it takes to finish the ringtone, otherwise it keeps starting the beginning of the ringtone).

Besides the `key="play_quick_reply_message",` entity works exactly in the same way and is already present in this integration.

Reply:
I think I agree with @edenhaus. Like, yes it's a nice way to give the user a quick way to play a chime, but _what is the use case?_ I would assume most people only visit the device page when they are setting up or checking something, not for actually doing actions.

I would argue that select entities would only be the best pick when it has a state. Otherwise a select entity could also be used as a remote, where all the buttons are put as options. While it works, its not ideal. So yea, I'd argue that playing a sound (and playing that quick text message) is an action and thus should be a service call (or buttons, but I think a service call is more flexible here when reolink adds more sounds or messages, otherwise you might have to add even more entities in the future)

Reply:
For the "quick replay play" I just added that entity below my camera stream so I can play a voice message when someone is at the door, no need to create something with services, just add the select entity.

Indeed for the play ringtone off the chime it is less likely a user wants to trigger it, but more likely to be used in automations and scripts, but if you use the select.select_option or a custom service really does not matter much.

Besides having it as an entity on the device page makes it a lot easier to find for users (in my opinion).

I agree buttons would be a even better fit, but you end up with way too many buttons....

I do like the select entity better, but if you ensist I will remove this entity and implement a service in a future PR.

Reply:
I removed it.